packages/web/src/pages/Flows/index.jsx

```diff
@@ -43,6 +43,7 @@ export default function Flows() {
 
     fetchData({ flowName, page });
 
+    setIsLoading(false);
```

Comment:
Let's consider using `onSettled` instead of `onSuccess` and achieve the same in the hook to fetch the data?